03-css-debugging/src/examples/examples.scss

```diff
@@ -0,0 +1,45 @@
+.Intro {
+  .first {
+    color: skyblue;
+  }
+  .second {
+    color: navy;
+  }
+  &__niceName {
+    color: maroon;
+  }
+}
+
+
+
+
+.Vertical {
+  vertical-align: center; // <-- doesn't work
+}
+
+
+
+
+.Z {
+  .Z__shouldBeOnTop {
+    z-index: 9999;
+  }
+  .Z__evilStackingContext {
+    position: relative;
+    display: inline-block;
+    &::after {
+      z-index: 1;
+      position: absolute;
+      top: -20px;
+      right: 0;
+      color: red;
+      content: 'X';
+      font-size: 2em;
+    }
+  }
+}
+
+
+#ddfdsfas, div[dsfas*=fsafdsa], .first.second, .dfasfsafasdfc {
```

Comment:
guess it is part of exercise..? :) as well commented code

Reply:
actually, this overwrite a style from `.Intro` because of non-unique naming (hiding .first.second between "unrelated" stuff to illustrate a more realistic situation that would be harder to spot) :)

and the commented code will be un-commented live to illustrate how to "debug" with backgrounds/outlines

I will add code comments in next PR, thank!!!